bundle.Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=machine-deletion
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.7.1+git
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.11.0+git
```

Comment:
Currently it is v1.25.1, isn't Ok?
https://github.com/medik8s/machine-deletion-remediation/blob/c14de13b378c08056c4b0cb5a9d1f1c249ffcddb/bundle.Dockerfile#L10